lib/ts/recipe/emailverification/emaildelivery/services/smtp/emailVerify.ts

```diff
@@ -0,0 +1,378 @@
+/* Copyright (c) 2021, VRAI Labs and/or its affiliates. All rights reserved.
```

Comment:
- The file structure of this folder is not the same as in other recipes.
- Missing SMTP service for this recipe?
- Missing index.ts file in services folder in this recipe?